adam/ontology/phase1_ontology.py

```diff
@@ -1396,34 +1400,62 @@ def _make_human_arm_segment():
 
```

Comment:
We should raise an issue to check all our defined schema to add in this `located` information with regards to left/right and front/back as many schema have distinctions like this which are not currently expressed.

Reply:
@LichMaster98 : I think we can continue to use #570 for that (which is why I was careful to say "enables #570" rather than "closes" in the PR description. :-) ).